modules/asm/main.tf

```diff
@@ -45,6 +45,7 @@ resource "kubernetes_config_map" "asm_options" {
 
   data = {
     multicluster_mode = var.multicluster_mode
+    ASM_OPTS          = var.enable_cni ? "CNI=on" : null
```

Comment:
nit: should this be 

```suggestion
    ASM_OPTS = var.enable_cni ? "CNI=on" : ""
```

Reply:
Do we want it to be set to `""` or excluded entirely? Usually I think excluding is better.

Reply:
When set to null the value is not applied to the configMap. Empty value could trigger unintended behavior.